lightning_pose/models/heatmap_tracker.py

```diff
@@ -260,12 +264,21 @@ def heatmaps_from_representations(
 
     def forward(
         self,
-        images: TensorType["batch", "frames", "channels":3, "image_height", "image_width"],
+        images: Union[TensorType["batch", "channels":3, "image_height", "image_width"],
+                      TensorType["batch", "views", "channels":3, "image_height", "image_width"]]
     ) -> TensorType["num_valid_outputs", "num_keypoints", "heatmap_height", "heatmap_width"]:
         """Forward pass through the network."""
         # we get one representation for each desired output.
-        representations = self.get_representations(images)
-        heatmaps = self.heatmaps_from_representations(representations)
+        shape=images.shape
+        if len(shape)>4:
```

Comment:
add a comment here just saying that if len(shape) > 4 we assume we have multiple views and need to combine images across batch/views before passing to network, then we reshape